warehouse/observations/tasks.py

```diff
@@ -51,7 +51,13 @@ def execute_observation_report(config: Configurator, session: SA_Session):
         config.task(report_observation_to_helpscout).delay(obj.id)
 
 
-@tasks.task(bind=True, ignore_result=True, acks_late=True)
+@tasks.task(
+    bind=True,
+    ignore_result=True,
+    acks_late=True,
+    autoretry_for=(Exception,),
```

Comment:
not strictly opposed to a catch all here, but is it worth being more restrictive?

Reply:
Good catch - I'll narrow it to only the ones we've seen so far - from remote services via `requests`.